pkg/storaged/dialog.jsx

```diff
@@ -626,7 +626,7 @@ export const SelectSpaces = (tag, title, options) => {
                         };
 
                         return (
-                            <li key={spc.desc} className="list-group-item">
+                            <li key={spc.block ? spc.block.Device : spc.desc} className="list-group-item">
```

Comment:
LGTM, and let's not forget that duplicate keys are not fatal here in production.